template-for-recording/record_screen_and_upload.bat

```diff
@@ -77,24 +87,32 @@ set PARAM_SOURCECODE_DIR=--sourcecode %APP_HOME%
 for /f "tokens=3" %%g in ('%JAVA_EXE% -version 2^>^&1 ^| findstr /i "version"') do (
     set JAVA_FULL_VERSION=%%g
 )
-set JAVA_FULL_VERSION=%JAVA_FULL_VERSION:"=%
+set "JAVA_FULL_VERSION=%JAVA_FULL_VERSION:"=%
```

Comment:
Same here

Reply:
We should not change the substitution syntax.
I think the problem is that %JAVA_FULL_VERSION% ends up not being defined.
Try this statement without defining JAVA_FULL_VERSION.

Reply:
I have tested with the not defined scenario and it works fine